README.md

```diff
@@ -21,39 +21,50 @@ how this growth and mortality are implemented can be found below.
 
 ## Quick Start
 
+This plot was generated using the following script as an example of how CoralBlox can be used:
+
+![coral blox plot](figures/plain_cb.jpg)
```

Comment:
I recommend figures be moved to an assets directory. It would be better as the package grows and evolves.

You can follow what other packages have done (e.g., https://github.com/rafaqz/Rasters.jl/blob/main/README.md?plain=1)